docs/configuration/configuration.md

```diff
@@ -773,6 +773,9 @@ region: <string>
 # instead be specified in the relabeling rule.
 [ port: <int> | default = 80 ]
 
+# The endpoint type to connect to. Must be one of public, admin or internal.
```

Comment:
If this is called availability, why do the docs say it's an endpoint type?

Reply:
I tried to incorporate your suggestions. is it fine now?